src/api-docs.js

```diff
@@ -3973,6 +3992,199 @@ const defaultApiDocs = {
+        summary: 'Load more addresses by pushing the end index of a wallet configured to index-limit scanning policy.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          },
+        ],
+        requestBody: {
+          description: 'Count of addresses to load.',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['count'],
+                properties: {
+                  count: {
+                    type: 'integer',
+                    description: 'Count of addresses to load.',
+                  },
+                }
+              },
+              examples: {
+                data: {
+                  summary: 'Load 10 more addresses.',
+                  value: {
+                    count: 10,
+                  }
+                }
+              }
+            }
+          }
+        },
+        responses: {
+          200: {
+            description: 'Success confirmation or handled error',
+            content: {
+              'application/json': {
+                examples: {
+                  success: {
+                    summary: 'Success',
+                    value: { success: true },
+                  },
+                  'wallet-not-ready': {
+                    summary: 'Wallet is not ready yet',
+                    value: { success: false, message: 'Wallet is not ready.', state: 1 }
+                  },
+                  'no-wallet-id': {
+                    summary: 'No wallet id parameter',
+                    value: { success: false, message: "Parameter 'wallet-id' is required." }
+                  },
+                  'invalid-wallet-id': {
+                    summary: 'Wallet id parameter is invalid',
+                    value: { success: false, message: 'Invalid wallet-id parameter.' }
+                  },
+                  'invalid-parameter': {
+                    summary: 'Invalid parameter',
+                    value: { success: false, error: [{ value: '"1"', msg: 'Invalid value', param: 'address', location: 'query' }] }
+                  }
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+    '/wallet/config/index-limit/last-index': {
+      post: {
+        summary: 'Set the last address index of a wallet configured to index-limit scanning policy.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          },
```

Comment:
[💬  comment] As this is a `js` document, why not parameterize this chunk of code that is repeating over the document?

Reply:
This is being done in #362

Reply:
Nice!